Nanoleaf.Client/Nanoleaf.Client/Models/Responses/Layout.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.ComponentModel;
+using Newtonsoft.Json;
+
+namespace Nanoleaf.Client.Models.Responses {
+	/// <summary>
+	/// Layout response returned from a layout request
+	/// </summary>
+	[Serializable]
+	public class Layout {
+		/// <summary>
+		/// Number of panels
+		/// </summary>
+		[JsonProperty] public int NumPanels { get; set; }
+		/// <summary>
+		/// Side of each length
+		/// </summary>
+		[JsonProperty] public int SideLength { get; set; } = 1;
+		/// <summary>
+		/// Array of position data
+		/// </summary>
+		[JsonProperty] public PanelLayout[] PositionData { get; set; } = Array.Empty<PanelLayout>();
+
+	}
+
+	/// <summary>
+	/// Layout info on a specific panel
+	/// </summary>
+	[Serializable]
+	public class PanelLayout {
```

Comment:
We need to use separate files for different classes. Let's follow the existing code design.